src/lib/remark-plugins/rewrite-tutorial-links/__tests__/rewrite-external-docs-link.test.ts

```diff
@@ -44,13 +43,7 @@ describe('rewriteExternalDocsLink', () => {
 			Object.keys(productSlugsToHostNames).map((productSlug: ProductSlug) => {
 				const hostname = `https://${productSlugsToHostNames[productSlug]}`
 				const productLandingPath = `/${productSlug}`
-
-				let expectedOutput
-				if (getIsBetaProduct(productSlug)) {
-					expectedOutput = productLandingPath
-				} else {
-					expectedOutput = undefined
-				}
+				const expectedOutput = productLandingPath
```

Comment:
Should we consider sentinel here?

Reply:
Or is this handled with the filtering out in the active product slugs from `lib/products`?

Reply:
I'm not sure 🤔 this is just test code and it doesn't have a learn landing page / any tests cases for it so it seems okay to not?

Reply:
Right, i'm just thinking in the link rewriter. I think since its using the active products in the link rewriter, sentinel links won't get rewritten.

Reply:
Ah, word. I don't think I'm using `activeProductSlugs` in the rewrite logic, but I can swap out the beta checks with that if you think that would be best? I'm just not sure in what cases sentinel links would come through in a way that they needed to be rewritten.